CHANGELOG-1.0.md

```diff
@@ -0,0 +1,138 @@
+# Release notes for 
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v0.5.0
+
+
+## Urgent Upgrade Notes 
+
+### (No, really, you MUST read this before you upgrade)
+
+- Do not call controller-expand if volume is in-use and can not be expanded while in-use. New RBAC rules for listing pods are required. ([#86](https://github.com/kubernetes-csi/external-resizer/pull/86), [@gnufied](https://github.com/gnufied))
```

Comment:
Can you prefix the note with "external-resizer will not call...". That way it won't sound like it's an action a user needs to take.

Reply:
Can you maybe combine this with the next one below? The new RBAC rules are not required if you disable the check.

Reply:
done.